Makefile

```diff
@@ -51,7 +52,7 @@ build/$(JATTACH): src/jattach/jattach.c
 
 build/$(PROFILER_JAR): src/java/one/profiler/*.java
 	mkdir -p build/classes
-	$(JAVAC) -source 6 -target 6 -d build/classes $^
+	$(JAVAC) -source ${JAVAC_RELEASE_VERSION} -target ${JAVAC_RELEASE_VERSION} -d build/classes $^
```

Comment:
Although there is no difference between `$()` and `${}` in Makefiles, please don't mix styles within a single line.

Reply:
ouch, right. I gonna fix. Please squash when merging this PR.